renderer/src/components/CharacterKillsCard.jsx

```
@@ -184,93 +254,73 @@
-        {error && (
-          <div className="mb-4 p-3 bg-red-100 text-red-700 rounded flex items-center">
-            <FaExclamationTriangle className="mr-2" />
-            <span className="text-sm">{error}</span>
-          </div>
-        )}
+          <button
+            onClick={() => triggerAggregation('weekly')}
+            disabled={aggregating}
+            title="Run Aggregation"
+            className="p-3 bg-purple-600 text-white rounded-lg hover:bg-purple-700 transition flex items-center justify-center disabled:opacity-50"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)